platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/Messages.java

```diff
@@ -72,6 +72,33 @@
   public static final String IMPORT_MISMATCH_DDF_HOME_ERROR =
       "Import error: mismatched ddf.home [%s]; expecting local system to be installed under [%s].";
 
+  public static final String IMPORT_KEY_OPEN_ERROR = "Import error: failed to open key file [%s].";
+
+  public static final String IMPORT_KEY_DECODE_ERROR = "Import error: could nod decode key [%s].";
+
+  public static final String IMPORT_KEY_PADDING_ERROR =
```

Comment:
Yeah I already wasn't a fan of the key padding and algorithm errors, since there should be no way of causing them to occur. The only one that should be possible to cause without making code changes is the `INVALID_KEY` error which could occur if the key was corrupted somehow, or if the zip was actually encrypted with a different key. I'm thinking i'll just change the padding and algorithm errors to log statements and keep them stored in the `ZipUtils` class instead

Reply:
Just have those 2 other exception bubble out as system errors so they end up being caught in the cfg mgr and log as internal errors